provider/src/batchProcessor.ts

```diff
@@ -5,6 +5,7 @@ import {
 } from './errorHandlers'
 import { processBatchRequests } from './requestProcessor'
 import { forwardRequestsToProvider } from './providerClient'
+import { storeENSResponseInKV } from './cache'
```

Comment:
On line 41 below, you call 
`await storeResponseInCache(request, chainId, response)`

Reply:
In that funcyiom, make sure you dont store requestId and json2.0